
---
bool is_plane_found
sensor_msgs/PointCloud2 cloud_plane
fkoat32[4]cloud_plane_coef
sensor_msgs/PointCloud2 cloud_cluster
geometry_msgs/PoseStamped point_goal
